Backpack/Classes/Panel/BPKPanel.m

```diff
@@ -0,0 +1,96 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#import "BPKPanel.h"
+
+#import <Backpack/Spacing.h>
+#import <Backpack/Color.h>
+#import <Backpack/Radii.h>
+
+NS_ASSUME_NONNULL_BEGIN
+@interface BPKPanel()
+@property (assign) UIEdgeInsets originalLayoutMargins;
+
+- (void)setup;
+@end
+
+
+@implementation BPKPanel
+
+- (instancetype)initWithFrame:(CGRect)frame {
+    return [self initWithPadded:YES];
```

Comment:
I think you should call. `[super initWithFrame:frame]`, so the frame property will be correctly set. This also removes confusion whether `iniWithPadded` calls any super initialisers

Reply:
Yeah good shout